checks/dangerous_workflow.go

```diff
@@ -196,9 +202,8 @@ func validateSecretsInPullRequests(workflow *actionlint.Workflow, path string,
 }
 
 func validateUntrustedCodeCheckout(workflow *actionlint.Workflow, path string,
-	dl checker.DetailLogger, pdata *patternCbData,
-) error {
-	if !usesEventTrigger(workflow, triggerPullRequestTarget) {
+	dl checker.DetailLogger, pdata *patternCbData) error {
```

Comment:
I find your version much more readable, but it might be a code-styling convention.
I'd install golang for vscode and apply its formatting just to make sure.